packages/swing-store-lmdb/src/lmdbSwingStore.js

```diff
@@ -370,10 +176,6 @@ function makeLMDBSwingStore(dirPath, forceReset, options) {
       txn.commit();
       txn = null;
     }
-    for (const fd of activeStreamFds) {
```

Comment:
`commit()` needs to commit the sqlite txn too, unless you're committing on every write (which would be ok).